examples/cqt2midi.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import division
+"""
+Utility function for converting an audio file
+to a MIDI file. Note that this method is nowhere close
```

Comment:
I think it is better to say "pretty_midi.PrettyMIDI object" here instead of "MIDI file" since that's really what you are demonstrating here; the CQT-to-MIDI is just a simple example of what you might do with this.  Know what I mean?  Same goes for the filename, in this case.